CHANGELOG.md

```diff
@@ -27,14 +27,16 @@ This release includes the following changes:
 - **(Breaking)** Removed `CallMessage.MessageType` and `CallMessage.ContentType` enumerations and types.
   Instead, those types have been replaced by `string`.
 
-The following is an example of the updated API:
+- **(Breaking)** Simplified the `Call` and `CallInvite` API for sending call messages. `Call.sendMessage` and `CallInvite.sendMessage` now take a plain-JS object, or interface, as a parameter.
+
+The following is an example of the updated API considering the above changes.
 ```ts
 const call = await voice.connect(...);
-call.sendMessage(new CallMessage({
+const outgoingCallMessage = await call.sendMessage({
```

Comment:
Why do we want to do it this way? Isn't this way different than the JS sdk?

Reply:
Doing it this way will actually match the JS SDK, instead of having a different API. In the JS SDK, we accept a plain JS object (a record) whereas the RN SDK without these changes will only accept an object.